packages/lumx-react/src/components/dialog/Dialog.tsx

```diff
@@ -75,6 +76,7 @@ const CLASSNAME = getRootClassName(COMPONENT_NAME);
  */
 const DEFAULT_PROPS: Partial<DialogProps> = {
     size: Size.big,
+    dialogProps: {}
```

Comment:
It should not be required. Spreading `undefined` works well

Reply:
Oops, I actually removed it and then did some rebase magic and it came back u_u.
Will remove.